docs/deployment/README.md

```diff
@@ -160,7 +168,12 @@ dev:                                # deployment name
 
 **Deploy `iam` stack**[^1]
 
-    $ ./node_modules/.bin/kes cf deploy --kes-folder iam --deployment <iam-deployment-name> --template node_modules/@cumulus/deployment/iam --region <region>
+```bash
+  $ DEPLOYMENT=<iam-deployment-name> \
```

Comment:
I added the same commands we use for deployment in our example/ directory to the template-deploy repo, so updating the documentation to use them because they seem easier, but I welcome feedback/thoughts

Reply:
That sounds good to me